client/src/app/pages/tasks/tasks-page.tsx

```diff
@@ -0,0 +1,348 @@
+  return (
+    <>
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">{t("titles.taskManager")}</Text>
+        </TextContent>
+      </PageSection>
+      <PageSection>
+        <div
+          style={{
+            backgroundColor: "var(--pf-v5-global--BackgroundColor--100)",
+          }}
+        >
+          <Toolbar {...toolbarProps} clearAllFilters={clearFilters}>
+            <ToolbarContent>
+              <FilterToolbar {...filterToolbarProps} />
+              <ManageColumnsToolbar
+                columns={columnState.columns}
+                setColumns={columnState.setColumns}
+                defaultColumns={columnState.defaultColumns}
+              />
+              <ToolbarItem {...paginationToolbarItemProps}>
+                <SimplePagination
+                  idPrefix="tasks-table"
+                  isTop
+                  paginationProps={paginationProps}
+                />
+              </ToolbarItem>
+            </ToolbarContent>
+          </Toolbar>
+
+          <Table {...tableProps} id="tasks-table" aria-label="Tasks table">
+            <Thead>
+              <Tr>
+                <TableHeaderContentWithControls {...tableControls}>
+                  {columnState.columns
+                    .filter(({ id }) => getColumnVisibility(id))
+                    .map(({ id }) => (
+                      <Th key={id} {...getThProps({ columnKey: id })} />
+                    ))}
+                  <Th width={10} />
+                </TableHeaderContentWithControls>
+              </Tr>
+            </Thead>
+            <ConditionalTableBody
+              isLoading={isFetching}
+              isError={!!fetchError}
+              isNoData={currentPageItems.length === 0}
+              noDataEmptyState={
+                <EmptyState variant="sm">
+                  <EmptyStateHeader
+                    titleText={t("message.noResultsFoundTitle")}
+                    headingLevel="h2"
+                    icon={<EmptyStateIcon icon={CubesIcon} />}
+                  />
+                </EmptyState>
+              }
+              numRenderedColumns={numRenderedColumns}
+            >
+              <Tbody>
+                {currentPageItems
+                  ?.map((task): [Task, { [p: string]: ReactNode }] => [
+                    task,
+                    toCells(task),
+                  ])
+                  .map(([task, cells], rowIndex) => (
+                    <Tr key={task.id} {...getTrProps({ item: task })}>
+                      <TableRowContentWithControls
+                        {...tableControls}
+                        item={task}
+                        rowIndex={rowIndex}
+                      >
+                        {columnState.columns
+                          .filter(({ id }) => getColumnVisibility(id))
+                          .map(({ id }) => (
+                            <Td key={id} {...getTdProps({ columnKey: id })}>
```

Comment:
This is the column id right?  The key should probably have the task.id in there as well.